.github/workflows/dotnet3.1Build.yml

```
@@ -0,0 +1,38 @@
+name: CodingGuidelines Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: 0 6 * * 1
+env:
+  version: 0.1.9
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: microsoft/setup-msbuild@v1.0.2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
+    - name: Use latest NuGet
+      uses: nuget/setup-nuget@v1
+    - name: Get Directory
+      run: pwd
+    - name: Dotnet Restore dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore
```

I would make this a single step.
```suggestion
    - name: Build
      run: dotnet build
```

---

We should also pass the version number in here. I believe you can do something like this:
```
dotnet build -p:Version="1.2.3"
```